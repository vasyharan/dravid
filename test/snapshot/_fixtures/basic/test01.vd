def foo(a, b, c) {
   val bar = (a + 10) * b - 1 / c
   bar
}

def main() {
  foo(1, 2, 3)
}