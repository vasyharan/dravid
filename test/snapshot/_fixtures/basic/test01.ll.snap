(keyword fn 1:0)
(id foo1 1:3)
(op ( 1:7)
(id a 1:8)
(op , 1:9)
(id b 1:11)
(op , 1:12)
(id c 1:14)
(op ) 1:15)
(op = 1:17)
(op { 1:19)
(keyword val 2:2)
(id d 2:6)
(op = 2:8)
(int 10 2:10)
(op * 2:13)
(id c 2:15)
(keyword if 3:2)
(id d 3:5)
(op { 3:7)
(op ( 4:4)
(id a 4:5)
(op + 4:7)
(int 10 4:9)
(op ) 4:11)
(op } 5:2)
(keyword elif 5:4)
(id a 5:9)
(op { 5:11)
(op ( 6:4)
(id a 6:5)
(op - 6:7)
(int 10 6:9)
(op ) 6:11)
(op } 7:2)
(keyword else 7:4)
(op { 7:9)
(op ( 8:4)
(id a 8:5)
(op * 8:7)
(int 10 8:9)
(op ) 8:11)
(op } 9:2)
(op } 10:0)
(keyword fn 12:0)
(id main 12:3)
(op ( 12:7)
(op ) 12:8)
(op = 12:10)
(op { 12:12)
(id foo 13:2)
(op ( 13:5)
(int 1 13:6)
(op , 13:7)
(int 2 13:9)
(op , 13:10)
(int 3 13:12)
(op ) 13:13)
(id foo1 14:2)
(op ( 14:6)
(int 1 14:7)
(op , 14:8)
(int 2 14:10)
(op , 14:11)
(int 3 14:13)
(op ) 14:14)
(id foo2 15:2)
(op ( 15:6)
(int 1 15:7)
(op , 15:8)
(int 2 15:10)
(op , 15:11)
(int 3 15:13)
(op ) 15:14)
(op } 16:0)
(keyword fn 18:0)
(id test1 18:3)
(op ( 18:8)
(id x 18:9)
(op ) 18:10)
(op = 18:12)
(int 1 18:14)
(op + 18:16)
(int 2 18:18)
(op + 18:20)
(id x 18:22)
(keyword fn 19:0)
(id test2 19:3)
(op ( 19:8)
(id x 19:9)
(op ) 19:10)
(op = 19:12)
(op ( 19:14)
(int 1 19:15)
(op + 19:16)
(int 2 19:17)
(op + 19:18)
(id x 19:19)
(op ) 19:20)
(op * 19:21)
(op ( 19:22)
(id x 19:23)
(op + 19:24)
(op ( 19:25)
(int 1 19:26)
(op + 19:27)
(int 2 19:28)
(op ) 19:29)
(op ) 19:30)
(eof 0:0)
